server/events/terraform/terraform_client.go

```diff
@@ -73,7 +74,8 @@ type DefaultClient struct {
 	// usePluginCache determines whether or not to set the TF_PLUGIN_CACHE_DIR env var
 	usePluginCache bool
 
-	terraformOutputChan chan<- *models.TerraformOutputLine
+	terraformOutputChan     chan<- *models.ProjectCmdOutputLine
+	ProjectCmdOutputHandler handlers.DefaultProjectCommandOutputHandler
```

Comment:
```suggestion
	projectCmdOutputHandler handlers.DefaultProjectCommandOutputHandler
```
replace every struct that uses terraformOutputChan with `	ProjectCmdOutputHandler handlers.DefaultProjectCommandOutputHandler`